android/PhysicalWeb/app/src/main/res/layout/fragment_nearby_beacons.xml

```diff
@@ -9,8 +9,7 @@
                     android:background="#ffffffff">
 
         <ListView
-            android:id="@android:id/list"
-            android:alpha="0"
+            android:id="@id/android:list"
```

Comment:
This was actually correct before; the id for the empty view should be changed to "@android:id/empty"